plugins/mcreator-localization/help/default/villagertrades/price2.md

```diff
@@ -0,0 +1,3 @@
+The second item the trade requires.
+
+NOTE: Optional
```

Comment:
Второй предмет, необходимый (если указан) для заключения сделки.

Reply:
I'd also shorten the original tip since making a NOTE for a single word is quite strange.
```suggestion
The second item required by the trade (if selected).
```

Reply:
No, second item is not required, why "if selected"?

Reply:
> No, second item is not required, why "if selected"?

It is required if the second item is selected